scanpy/plotting/_anndata.py

```diff
@@ -1709,18 +1709,23 @@ def dotplot(
     num_categories: int = 7,
     expression_cutoff: float = 0.0,
     mean_only_expressed: bool = False,
-    color_map: str = 'Reds',
+    color_map: str = 'RdBu_r',
+    style: Optional[str] = 'square color',
```

Comment:
I don't think the default should change until 2.0. I'm also not sure I like the square version better.

Reply:
Thanks, indeed I wanted to set that back as in the original case.